src/main/java/com/example/books/model/Role.java

```diff
@@ -24,4 +24,8 @@ public class Role {
     @Enumerated(value = EnumType.STRING)
     @Column(unique = true)
     private RoleName name;
+
+    public Role() {
+
+    }

```

Comment:
@ NoArgsContructor

Reply:
Is it big difference? Why should i use annotation instead of defining it like that?